ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -254,6 +254,17 @@ def einsum(equation, *inputs, **kwargs):
 def adjoint(matrix, name=None):
     return ivy.adjoint(matrix)
 
+# qr
+@to_ivy_arrays_and_back
+def qr(input, full_matrices=False, name=None):
+    if full_matrices:
+        Q, R = ivy.qr(input)
+        Q = []
+        ret = Q, R
+    else:
+        ret = ivy.qr(input)
+    return ret
+=======
```

Comment:
Hi @saravanadivakaruni 

I'm not sure what you're doing wrong here but these `=====` lines are still showing up in the modified files. Could you kindly remove these lines and make a new commit so that they are removed?